test/resources/complex_xml_transformed.yml

```diff
@@ -0,0 +1,626 @@
+﻿---
+- - - ECosdXmlRecord
+    - nhsnumber: '1111111111'
+      dateofbirth: '2023-10-26'
+      orgidecodofpro: FWC
+      surname: 0PkEC9DE4m3mPIoaBUyrNxCa4pxqRwLuyx0
+      forenames: 6mt6AcIOFTMqynaJK8iYuFNYVlrCFaRgtKL
+      postcode: WXO9Epup
+      sex: '9'
+      sexual_orientation_at_diag: U
+      gpcode: DuyujYLZ
+      gppracticecode: UYDN5C
+      previoussurname: I2mgthmNL9zCbUI1w78H9O4csujfww4ADVm
+      ethniccategory: '99'
+      primarydiagnoses: 5vTddo
+      tumourlaterality: '9'
+      diagnosisdate: '2023-10-26'
+      :rawtext:
+        cosdrecordid: AAD0ce33-d168-26c6-A4Df-7A0B4d2a5aeC
+        nhsnumber: '1111111111'
+        hospitalnumber: ''
+        nhsnumberstatusindicatorcode: '04'
+        dateofbirth: '2023-10-26'
+        hospitalnumberprovidercode: FWC
+        surname: 0PkEC9DE4m3mPIoaBUyrNxCa4pxqRwLuyx0
+        forenames: 6mt6AcIOFTMqynaJK8iYuFNYVlrCFaRgtKL
+        address: ''
+        postcode: WXO9Epup
+        sex: '9'
+        sexual_orientation_at_diag: U
+        gpcode: DuyujYLZ
+        gppracticecode: UYDN5C
+        previoussurname: I2mgthmNL9zCbUI1w78H9O4csujfww4ADVm
+        ethniccategory: '99'
+        primarydiagnoses: 5vTddo
+        tumourlaterality: '9'
+        diagnosisdate: '2023-10-26'
+        address_1: Address line 1
+        address_2: Address line 2
+        address_3: Address line 3
+        address_4: Address line 4

```

Comment:
Is it expected behaviour to have an empty `address` field, as well as populated `address_1` .. `address_4` in the `rawtext`?

Reply:
Yes, there are two column mappings for `address`. `StructuredAddress` is present in the data and has spawned the incremented versions. `UnstructuredAddress` is not in the data so the rawtext key is blank, as per standard ndr_import behaviour. We could probably map these to different `rawtext_name`s, to avoid any confusion

Reply:
If it's doing the right thing, I'm OK with the result.